practico_01/ejercicio-05.py

```diff
@@ -3,5 +3,10 @@
 
 
 # Resolver utilizando listas y el operador in.
+vocales = "aeiou"
```

Comment:
No usar variables globales

Reply:
Si no lo pongo como variable tendría q hacer
`return letra in "aeiou"`
...que seria lo mismo, no?
No estoy entendiendo.

Reply:
Exacto, pero tendrías que hacer que funcione para minúsculas también

Ojo, el problema no era usar variables, era usar variables **globales**